src/main/java/net/atos/zac/zoeken/model/ZaakIndicatie.java

```diff
@@ -17,5 +17,7 @@ public enum ZaakIndicatie {
     HEROPEND,
     HOOFDZAAK,
     DEELZAAK,
-    VERLENGD;
+    VERLENGD,
+    INTAKE,
```

Comment:
Is this the Dutch name for it? Just wondering...

Reply:
The meaning here should be "extended". Would appreciate help for making it correct in Dutch

Reply:
The DOC in the issue says:

> verlengen
> X (zaak open, niet heropend, niet opgeschort, en niet al keer verlengd)

Reply:
Extended = verlengd.

We are taling about `INTAKE` no?

Reply:
yes, the question is about INTAKE - I wondered if that was the right term.

Reply:
Good point. We should preferable only use terms that are used in the UI or elsewhere in ZAC. Is this the case for `INTAKE`?

Reply:
Well INTAKE is met across the code, but we can definitely change it here

Reply:
Looking at app/src/assets/i18n/en.json and app/src/assets/i18n/nl.json we use `intake` in both languages. 

Not sure why, but it seems to be the case.